examples/generative/corrdiff/conf/config_train_base.yaml

```diff
@@ -76,12 +76,8 @@ wandb_mode: online
   # Weights & biases mode [online, ofline, disabled]
 wandb_project:  "Modulus-Generative"
  # Weights & biases project
-wandb_group: "CorrDiff-DDP-Group"
- # Weights & biases group
-wandb_entity: "CorrDiff-DDP-Group"
+wandb_entity: "CorrDiff"
```

Comment:
Is this actually the name of the "entity" for corrdiff? In the link below, "entity" is "nv-research-climate". Maybe we can just avoid setting an option here and let wandb do it's default behavior (entity=user name).

https://wandb.ai/nv-research-climate/edm-chaos/runs/jfw64cdl/overview?nw=nwusernbren12

Reply:
I think we will be creating a new wandb team, and can edit these configs based on that.